docs/CTPUSHTEMPLATESCHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CleverTap Push Templates SDK CHANGE LOG
 
+### Version 1.0.5.1 (March 14, 2023)
```

Comment:
@piyush-kukadiya need to update main push template changelog post release. Create a separate PR just for changelog changes